docs/PrinciplesReviewWorkflow.md

```diff
@@ -3,16 +3,16 @@ layout: doc
 title: Principle Review Process
 ---
 
-This page describes the process for reviewing, updating, and creating [OBO Foundry Principles](http://obofoundry.github.io/principles/fp-000-summary.html).
+This page describes the process for reviewing, updating, and creating [OBO Foundry Principles](http://obofoundry.github.io/principles/fp-000-summary.html) by the OBO Foundry Editorial Working Group.
 
-- Discuss the principle on working group call
+- Discuss the principle during a working group call
 - Write up proposed revisions as a GitHub issue (not a pull request)
 - Inform the Editorial WG via email that the ticket needs discussion (if it doesn’t happen automatically)
```

Comment:
If what doesn't happen automatically? The informing, or the discussing?

Reply:
At the time this was written, it wasn't clear if there was some automatic message sent to members. Since it is now clear that this does not happen, I removed the phrase.